contracts/programs/ocr2/src/lib.rs

```diff
@@ -703,7 +703,7 @@ fn transmit_impl<'info>(ctx: Context<Transmit<'info>>, data: &[u8]) -> Result<()
     state.config.latest_transmitter = ctx.accounts.transmitter.key();
 
     // calculate and pay reimbursement
-    let reimbursement = calculate_reimbursement(report.juels_per_lamport, signature_count)?;
+    let reimbursement = calculate_reimbursement(report.juels_per_lamport, signature_count)?; // gjuels
```

Comment:
Can we rename `reimbursement` to `reimbursement_gjuels`?
Similarly, `calculate_reimbursement` -> `calculate_reimbursement_gjuels`?

Reply:
(Ditto for other variables in the code that store a value with a unit)

Reply:
👍🏼